### PR TITLE
🔀 ✨ [hk] added hotkey for collecting easter eggs 

### PR DESCRIPTION
**Pull request by aallen92**
**What kind of update does this PR provide?** *Please check*

- [ ] new translations / updated translations / translation fixes
- [x] a new feature
- [ ] a new module
- [ ] a bugfix for an existing feature / module
- [ ] improvement of style or user experience
- [ ] other: *Please fill out*

**Which language(s) did you add/update translations for?**
*use the xx_XX notation for exact language!*
* cs_CZ
* de_DE
* en_AU
* en_GB
* en_US
* es_ES
* fi_FI
* fr_FR
* nb_NO
* nl_NL
* pl_PL
* sv_SE

**What is included in your update?**
* Added hotkey for collecting easter eggs
* Added translation for all existing hotkey translations

**Additional notes**
Please add any further notes here!

### CODE REVIEW

**Comment by jxn-30** [comment]
@LSS-Manager deploy

File: src/modules/hotkeys/i18n/en_AU.root.json [hunk 1]
```diff
@@ -130,6 +130,7 @@
                 "onlyAmbulance": "Backalarm only ambulances",
                 "title": "Backalarm"
             },
+            "easter_click": "Collect Easter Egg",
```
Comment (jxn-30) [inline src/modules/hotkeys/i18n/en_AU.root.json:133]: I'm not yet sure if using the "Easter egg"-Term only.
Maybe we can find something general without listing each kind (sunflower, pumpkin, snowman, heart).

How about something like "Collect Easter Egg (or similar collective)"?
For DE I could think of using "Sammelobjekt (z.B. Osterei) einsammeln".

But I'm not yet sure. Maybe there are further ideas / opinions @LSS-Manager/developers ?

Comment (aallen92) [reply src/modules/hotkeys/i18n/en_AU.root.json:133]: How about Collect Seasonal Items? It's a fair point it doesn't signal well I guess that it works on other collectibles.

Comment (aallen92) [reply src/modules/hotkeys/i18n/en_AU.root.json:133]: Requested a review by accident - mistook the button for a refresh button 🤦🏻‍♂️

**Comment by Cr4zyc4k3** [comment]
I'm not quite sure whether this is allowed by the ToS

**Comment by jxn-30** [comment]
> I'm not quite sure, whether this is allowed by ToS.

where is the difference between pressing a key on your keyboard and using a mouse click? That would basically make the whole module be agains ToS as it mostly is only a "click a button / element when pressing this key".

File: src/modules/hotkeys/i18n/en_AU.root.json (hunk 1 above)
Comment (Cr4zyc4k3) [reply src/modules/hotkeys/i18n/en_AU.root.json:133]: Agree, we should mention an example. Especially for new players, which never had an event.

**Comment by Cr4zyc4k3** [comment]
Eg. the easter eggs are "hidden" on the whole page. It would destroy the whole "searching experience" (Dunno how to call it) if you just need to press a button instead of "searching" for the eggs.

**Comment by aallen92** [comment]
> Eg. the easter eggs are "hidden" on the whole page. It would destroy the whole "searching experience" (Dunno how to call it) if you just need to press a button instead of "searching" for the eggs.

I guess I felt ok with it because they are always in the same position next to the mission title. Spotting it and pressing a hotkey felt the same as spotting it and moving the mouse to click it. That said you guys will know the ToS better than I do!

File: src/modules/hotkeys/i18n/en_AU.root.json (hunk 1 above)
Comment (jxn-30) [reply src/modules/hotkeys/i18n/en_AU.root.json:133]: I am fine with "Collect Seasonal Items e.g. Easter Egg"!
de_DE: "Saisonales Sammelobjekt einsammeln (z.B. Osterei)"

**Comment by jxn-30** [comment]
In my eyes it would be fine with ToS but let's put this on hold and ask the game devs, just to be sure!

**Comment by deoxis9001** [comment]
fr_FR:
"Collecter un objet saisonnier (par ex. un œuf de Pâques)"

**Comment by jxn-30** [comment]
> In my eyes it would be fine with ToS but let's put this on hold and ask the game devs, just to be sure!

The team confirmed that this hotkey is fine!

**Comment by aallen92** [comment]
> The team confirmed that this hotkey is fine!

That's good news! I'll try and get the amendments to the label done in the next day or so in that case

**Review by jxn-30 — APPROVED** [review]

**Pull request merged**